packages/shared/src/components/profile/devcard/DevCardStats.tsx

```diff
@@ -28,6 +30,12 @@ export function DevCardStats({
         iconClassName="text-raw-onion-40"
         Icon={ReputationIcon}
       />
+      <DevCardStatsSection
+        amount={longestStreak}
+        label="Longest streak"
+        iconClassName="p-[0.19rem]"
```

Comment:
This is added because this icon is different from others, while others do seem to have some padding incorporated in the SVG this one doesn't making it bigger in the same sizes.
I tried changing the icon, but that would mean changing a lot into where this icon is used as well, like the streaks button on header, the modals, the completed streak on the popup, and still the sizes wouldn't fit the same as they do now in the current usage, so decided to just move forward for now.

It's an issue though I've seen sometime with icons we have where some are built differently with this padding or not, and in designs this padding is sometimes considered and sometimes not.

Reply:
It's better to check with design team, generally speaking all the outline of icons should be the same size.
It happened before where they can easily fix this.

Reply:
They can fix it true, we can even fix it without them if we export the svg they are using now for this, but that would mean we need to change all other usages of it and make it align.